gef.py

```diff
@@ -5484,8 +5484,13 @@ def set_fs(uc, addr):    return set_msr(uc, FSMSR, addr)
     set_gs(emu, SEGMENT_GS_ADDR)
 """
 
+        try:
```

Comment:
This part is untested. I currently don't have unicorn installed.

Reply:
Actually the `try`/`except` is not even necessary: if `python` is not in the PATH, we want a noisy error for the user.

Reply:
Fixed now.